asapdiscovery-ml/asapdiscovery/ml/config.py

```diff
@@ -138,22 +138,22 @@ class EarlyStoppingConfig(BaseModel):
         ),
     )
     # Parameters for best
-    patience: int = Field(
+    es_patience: int = Field(
```

Comment:
I'd rather leave these names as they are, since I think it makes more sense to not have the `es_` prefix when we're already in the `EarlyStoppingConfig`. I think my preferred solution would just be to change the `es_config` dicts in `cli.py` to match the ones in the methods for the other models (see my note in  `cli.py` as well)

Reply:
Should we change `es_type` to `type` for consistency? It was `es_type` before I edited

Reply:
I think leave the `es_type` as is, since that's what we have in the other `Config` classes. even though it's technically fine, it feels weird to me to have a field that has a protected name (`type`) which is why I like the `..._type` pattern